.eslintrc.js

```diff
@@ -180,6 +181,7 @@ module.exports = {
         "jest/no-hooks": OFF,
         "jest/no-mocks-import": OFF,
         "jest/prefer-expect-assertions": OFF,
+        "jest/unbound-method": OFF,
```

Comment:
Is this error expected when the rule is turned on?

<img width="529" alt="Screen Shot 2023-01-03 at 12 01 41 PM" src="https://user-images.githubusercontent.com/47064971/210404801-2e3263b2-4195-4ce6-8764-9a7842381a4f.png">

Reply:
Fixed it.